test/JDBC/expected/dbcreator_role-vu-verify.out

```diff
@@ -103,6 +115,18 @@ dbcreator_login1
 ~~END~~
 
 
+-- should not have membership in database guest role
+SELECT r.rolname AS parent_role, mr.rolname as member 
+FROM pg_auth_members m 
+JOIN pg_roles r ON (m.roleid = r.oid) 
+JOIN pg_roles mr ON (m.member = mr.oid) 
+where mr.rolname = 'dbcreator';
+go
+~~START~~
+varchar#!#varchar
+~~END~~
+
+
```

Comment:
Please also include test cases like 
login is member of db_creator ....  login has a mapped user in a database and login is not able to select on tables that are granted to guest but not to its mapped user.

Reply:
If we don't have membership in guest that will result in not granting but will add the testcase.